### PR TITLE
Module 12

### PR DESCRIPTION
**Pull request by A-J1993**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -129,4 +129,11 @@ geckodriver.exe
 geckodriver.log
 
 #Chrome Driver
-chromedriver.exe
\ No newline at end of file
+chromedriver.exe
+
+#Terraform_files
+.terraform.lock.hcl
```
Comment (hugh-emerson) [inline .gitignore:135]: The lock file should actually be committed. It's needed to keep versions of plugins correct... similar to the role of poetry.lock.

File: variables.tf [hunk 2]
```diff
@@ -0,0 +1,27 @@
+
+variable "location" {
+ description = "The Azure location where all resources in thisdeployment should be created"
+ default = "uksouth"
+}
+variable "SECRET_KEY" {
+    default = "secret-key"
```
Comment (hugh-emerson) [inline variables.tf:7]: This should also be sensitive. It's used to encrypt the session cookie, so if it leaked anyone could pretend to be logged into your to do app.

**Review by hugh-emerson — APPROVED** [review]
Looks good. A couple of minor comments but merge when you're ready. 👍

File: .gitignore (hunk 1 above)
Comment (A-J1993) [reply .gitignore:135]: Removed from here and Dockerignore

File: variables.tf (hunk 2 above)
Comment (A-J1993) [reply variables.tf:7]: Value moved to `terraform.tfvars` which will be ignored.

File: .gitignore (hunk 1 above)
Comment (A-J1993) [reply .gitignore:135]: (In the upcoming Pull request)

File: variables.tf (hunk 2 above)
Comment (A-J1993) [reply variables.tf:7]: (In the upcoming Pull request)

**Pull request merged**